src/store/epics/transfer.ts

```diff
@@ -110,9 +121,20 @@ export const transactionSuccess$: RootEpic = action$ =>
     ),
   )
 
-export const transactionFailed$: RootEpic = action$ =>
+export const transactionFailed$: RootEpic = (action$, state$, { sendAnalyticsEvent }) =>
   action$.pipe(
     filter(isActionOf(transactionFailed)),
+    withLatestFrom(state$),
+    map(([_, state]) =>
```

Comment:
maybe a little destructuring would be nice here to make it clear what we're looking for and also reduce the depth of nested property accessing the following calls eg.

```typescript
      map(([_, { accounts, connections, transfer }) =>
```